modules/logging-loki-retention.adoc

```diff
@@ -92,9 +93,11 @@ spec:
 <1> Sets retention policy by tenant. Valid tenant types are `application`, `audit`, and `infrastructure`.
```

Comment:
How about use one type of secret (s3,logging-loki-s3)  in all pages.

Reply:
Changed, lgtm please?